modules/ROOT/pages/cassandra-data-migrator.adoc

```diff
@@ -1,30 +1,14 @@
-= {cstar-data-migrator}
+= Introduction to {cstar-data-migrator}
```

Comment:
Remove "Introduction to". This page ends up including everything about CDM, so it's not just an intro.